cli/args/config_file.rs

```diff
@@ -666,6 +669,39 @@ impl ConfigFile {
     self.json.import_map.clone()
   }
 
+  /// This function prints a warning if "importMap" is used in the config file
+  /// and either "imports" or "scopes" configuration are specified.
+  pub fn to_import_map(&self) -> Option<serde_json::Value> {
+    fn print_warning_if_needed(maybe_import_map_path: Option<&str>) {
+      if maybe_import_map_path.is_some() {
+        log::warn!("{} \"importMap\" setting is ignored when \"imports\" or \"scopes\" are specified in the config file.", colors::yellow("Warning"));
+      }
+    }
+
+    match (&self.json.imports, &self.json.scopes) {
+      (Some(imports), Some(scopes)) => {
+        print_warning_if_needed(self.json.import_map.as_deref());
+        Some(json!({
+          "imports": imports.clone(),
+          "scopes": scopes.clone(),
+        }))
+      }
+      (Some(imports), None) => {
+        print_warning_if_needed(self.json.import_map.as_deref());
+        Some(json!({
+          "imports": imports.clone(),
+        }))
+      }
+      (None, Some(scopes)) => {
+        print_warning_if_needed(self.json.import_map.as_deref());
+        Some(json!({
+          "scopes": scopes.clone(),
+        }))
+      }
+      _ => None,
+    }
+  }
```

Comment:
Does something like this work?

```suggestion
    if self.json.imports.is_some() || self.json.scopes.is_some() {
        print_warning_if_needed(self.json.import_map.as_deref());
        Some(json!({
          "imports": self.json.imports.clone(),
          "scopes": self.json.scopes.clone(),
        }))
    } else {
      None
    }
```